### ISSUE TITLE
Constant training loss in the model adapter card

### ISSUE THREAD

**Issue by Michelet-Gaetan** [huggingface/alignment-handbook#192]
Hello,

I could fine-tune a model using a small dataset and I see that the validation loss decreases, while the training loss remains the same in the model card.

I don't think this is normal, even though the new task I try to teach the model is similar to what it already does, I think it should be able to learn from the dataset. I took a look at the trainer_state.json file created during the fine-tuning process and I saw that the training_loss for step 2 is different from the one displayed in the model card.

**Results from model_card:**

|Training Loss  |      Epoch 	|  Step 	 | Validation Loss|
|-------|-------|-------|-------|
|1.3185 	          |     1.0 	|          1   |	  1.4256|
|1.3185 	          |     1.1429 	|   2 	 |         1.3196|

**Results from the trainer_state.json:**

"log_history": [
    {
      "epoch": 1.0,
      "grad_norm": 1.1992276906967163,
      "learning_rate": 0.0002,
      "loss": 1.3185,
      "step": 1
    },
    {
      "epoch": 1.0,
      "eval_loss": 1.4256268739700317,
      "eval_runtime": 1.7474,
      "eval_samples_per_second": 1.145,
      "eval_steps_per_second": 0.572,
      "step": 1
    },
    {
      "epoch": 1.1428571428571428,
      "eval_loss": 1.3196333646774292,
      "eval_runtime": 1.552,
      "eval_samples_per_second": 1.289,
      "eval_steps_per_second": 0.644,
      "step": 2
    },
    {
      "epoch": 1.1428571428571428,
      "step": 2,
      "total_flos": 823612516859904.0,
      "train_loss": 0.7439389228820801,
      "train_runtime": 27.974,
      "train_samples_per_second": 0.5,
      "train_steps_per_second": 0.071
    }

Does the training loss remain the same, or is there a problem with the model card generation?

Have a nice day!

**Comment by Michelet-Gaetan** [comment]
My bad, due to the small number of samples, the process was done in 2 steps. 

My logging_step variable was set to 5, which explains why there is a value missing in the log_history list for step 2. When reducing the logging_step to 1, a new value is added to this list, and the model card presents two different training_loss values (decreasing).
